besser/bot/platforms/websocket/websocket_platform.py

```diff
@@ -131,6 +135,41 @@ def reply(self, session: Session, message: str) -> None:
         payload = Payload(action=PayloadAction.BOT_REPLY_STR,
                           message=message)
         self._send(session.id, payload)
+        
+    def reply_file(self, session: Session, file_path: str = None, file_data: bytes = None, file_name: str = None, file_type: str = None, 
+                   file_base64: dict = None) -> None:
+        """A bot file message (usually a reply to a user message) is sent to the session platform to show it to the user.
```

Comment:
Same as in telegram_platform:

replace "is sent to the session platform to show it to the user."
by: "is sent to the user"

(The session platform is the one sending the file)